MarkBind/markbind
# Fix incorrect locale examples in user guide

**What is the purpose of this pull request?**




**Overview of changes:**
Fixes #1761
- Update locale examples to the correct version (replacing `_` with `-`)
- Minor formatting update on the locale section in UG

**Anything you'd like to highlight/discuss:**
nil

**Testing instructions:**
Add one of the example locales in a `site.json` as 
```
"locale": "sq-AL"
```
which should work if you run `markbind serve` ("sq_AL" will not)



**Proposed commit message: (wrap lines at 72 characters)**


Fix incorrect locale examples in user guide
---

**Checklist:** :ballot_box_with_check:
